src/rules/no-extraneous-dependencies.js

```diff
@@ -200,6 +204,7 @@ module.exports = {
           'peerDependencies': { 'type': ['boolean', 'array'] },
           'bundledDependencies': { 'type': ['boolean', 'array'] },
           'packageDir': { 'type': ['string', 'array'] },
+          'ignore': { 'type': 'regexp' },
```

Comment:
it's a very bad idea to allow regex strings in eslint configs; that's how you get CVEs. this should be a glob string instead.

i'd also expect it to be allowed to be an array of glob strings.

Reply:
Thanks, that's a good idea!